shared/src/ledger/events.rs

```diff
@@ -151,7 +162,7 @@ impl From<ProposalEvent> for Event {
 }
 
 /// Convert our custom event into the necessary tendermint proto type
-impl From<Event> for crate::facade::tendermint_proto::abci::Event {
+impl From<Event> for tendermint_proto::abci::Event {
```

Comment:
@tzemanovic this should be `crate::facade::tendermint_proto::abci::Event`, sorry

Reply:
it affects abcipp build right? We should be maybe check it in the CI

Reply:
yep